kytos/core/log_manager.py

```diff
@@ -0,0 +1,85 @@
+"""Module with classes used to handle logs displayed by Kytos SDN Platform."""
+import logging
+from logging.handlers import SysLogHandler
+
+__all__ = ('LogManager')
+
+
+class LogManager:
+    """Class responsible to handle all logs send by kytos and the Napps."""
+
+    fmt = '%(asctime)s - %(levelname)s [%(name)s] (%(threadName)s) %(message)s'
+    handlers_enabled = []
+    loggers = {}
+
+    def __init__(self, mode='INFO'):
+        """Contructor of LogManager.
+
+        This method will create a LogManager instance with the mode given.
+
+        Args:
+            mode(string): Type of log that will be registered by kytos.
+        """
+        logging.basicConfig(format=self.fmt, level=getattr(logging, mode))
+
+    def new_logger(self, logger_name=None):
```

Comment:
What if no `logger_name` is passed? It may be better to set this attribute (`logger_name`) as mandatory.